commands/wip.command.js

```diff
@@ -1,37 +1,58 @@
-const { addLabel, removeLabel } = require('../helpers/label.helper');
-const { listIssueLabels } = require('../helpers/listLabels.helper');
+const { addLabel, removeLabel } = require("../helpers/label.helper");
+const { listIssueLabels } = require("../helpers/listLabels.helper");
 
 const WIPCommand = async context => {
 	const { title } = context.payload.issue || context.payload.pull_request;
 	const labelsFromIssues = await listIssueLabels(context);
 
 	const wipLabel = labelsFromIssues.data.find(
-		label => label.name === ':construction: WIP'
+		label => label.name === ":construction: WIP"
+	);
+	const foundReadyForReviewLabel = labelsFromIssues.data.find(
+		label => label.name === ":mag: Ready for Review"
 	);
 
 	if (
-		title.includes('WIP') ||
-		title.includes('Work In Progress') ||
-		title.includes('work in progress') ||
-		title.includes(':construction:')
+		title.includes("WIP") ||
+		title.includes("Work In Progress") ||
+		title.includes("work in progress") ||
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹` with `····`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060852)
```suggestion
    title.includes("work in progress") ||
```